pkg/kds/server/grpc.go

```diff
@@ -0,0 +1,64 @@
+package server
+
+import (
+	"fmt"
+	"net"
+
+	mesh_proto "github.com/Kong/kuma/api/mesh/v1alpha1"
+	kds_config "github.com/Kong/kuma/pkg/config/kds"
+
+	"google.golang.org/grpc"
+
+	"github.com/Kong/kuma/pkg/core"
+	"github.com/Kong/kuma/pkg/core/runtime/component"
+)
+
+const grpcMaxConcurrentStreams = 1000000
```

Comment:
I guess an arbitrary number, but still - do we have a proof this can be supported in practice?

nit: I also prefer these numbers to be in the form `1*1000*1000` so one can read it easier.

Reply:
This number is taken from another xDS implementations:
- `pkg/sds/server/grpc.go`
- `pkg/mads/server/grpc.go`
- `pkg/xds/server/grpc.go`

All of them have the same constant with the same value. @jakubdyszkiewicz do you have a notion where is that number taken from?

Reply:
Out of the thin air ;)

You have to pick a number, measure with metrics it and adjust if needed. This was our pick.